src/Components/Components/src/LayoutComponentBase.cs

```diff
@@ -17,5 +19,20 @@ public abstract class LayoutComponentBase : ComponentBase
         /// </summary>
         [Parameter]
         public RenderFragment? Body { get; set; }
+
+        /// <inheritdoc />
+        public override Task SetParametersAsync(ParameterView parameters)
```

Comment:
Is this really the simplest way to do it? It has a runtime perf implication. Really minor, but still it's weird to be imposing that to fix a compile-time linker issue.

Is it possible to apply some attribute to `Body { set; }` to tell it never to be trimmable?

Reply:
I think we could apply the `DynamicDependency` attribute to either the constructor or `SetParametersAsync` to say preserve all properties. Alternatively you could use a trimmer descriptor for this type. I have a feeling the trimmer folks would like us to avoid the latter.